### ISSUE TITLE
How to set default values (from) and set selected ones to the parent scope ?

### ISSUE THREAD

**Issue by itamair** [mapeveri/ionic-multiselect#17]
Tnx for this nice directive ... but
May be I am not getting the evidence, but I don't figure out how to pass default values at the first load, from the parent scope.

**Comment by mapeveri** [comment]
This is not good for you? https://github.com/mapeveri/ionic-multiselect/blob/master/example/controller.main.js

**Comment by itamair** [comment]
Ok ... I got it.
I needed to set default checked/selected values, and to do that I have finally realized that should be set as "checked=true" the property of default values.
The documentation is not clear at all on this (imho) and might be cleared out / detailed better (to make it easier the use of the directive, especially from the less skilled ones, like me).
Besides this (moreover) the second documented markup example (with this attribute definition: checked-property="selected") seems isn't working ... and brought me in the wrong way at the beginning.
